src/containers/layout/account-menu/AccountMenu.tsx

```diff
@@ -1,69 +1,60 @@
 import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
-import { Link } from 'react-router-dom'
-import MenuItem from '@mui/material/MenuItem'
-import { MenuProps } from '@mui/material'
+import { useNavigate } from 'react-router-dom'
 import LogoutIcon from '@mui/icons-material/Logout'
 
 import AppMenu from '~/components/app-menu/AppMenu'
 import { styles } from '~/containers/layout/account-menu/AccountMenu.styles'
 import { useAppSelector } from '~/hooks/use-redux'
 
 import { authRoutes } from '~/router/constants/authRoutes'
-import { spliceSx } from '~/utils/helper-functions'
 import { RouteItem } from '~/types'
 
 interface AccountMenuProps {
-  anchorEl: MenuProps['anchorEl']
+  anchorEl: HTMLElement | null
   onClose: () => void
 }
 
 const AccountMenu: FC<AccountMenuProps> = ({ anchorEl, onClose }) => {
```

Comment:
```suggestion
const AccountMenu: React.FC<AccountMenuProps> = ({ anchorEl, onClose }) => {
```
I saw that correct to write FC with this method in Quiz PR, i think this is minor, but add please, for better code style

Reply:
@Iarynovskyi added update